<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWaveLine</class>
 <widget class="QWidget" name="frmWaveLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="WaveLine" name="widget1">
     <property name="bgColorStart">
      <color>
       <red>60</red>
       <green>60</green>
       <blue>60</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveLine" name="widget2">
     <property name="bgColorStart">
      <color>
       <red>60</red>
       <green>60</green>
       <blue>60</blue>
      </color>
     </property>
     <property name="lineColor">
      <color>
       <red>255</red>
       <green>107</green>
       <blue>107</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WaveLine" name="widget3">
     <property name="bgColorStart">
      <color>
       <red>60</red>
       <green>60</green>
       <blue>60</blue>
      </color>
     </property>
     <property name="lineColor">
      <color>
       <red>24</red>
       <green>189</green>
       <blue>155</blue>
      </color>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveLine</class>
   <extends>QWidget</extends>
   <header>waveline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
